packages/flutter/lib/src/widgets/widget_state.dart

```diff
@@ -424,7 +424,7 @@ class _WidgetStateBorderSide extends WidgetStateBorderSide {
 ///  * [ShapeBorder] the base class for shape outlines.
 ///  * [MaterialStateOutlinedBorder], the Material specific version of
 ///    `WidgetStateOutlinedBorder`.
-abstract class WidgetStateOutlinedBorder extends OutlinedBorder implements WidgetStateProperty<OutlinedBorder?> {
+abstract class WidgetStateOutlinedBorder extends LinearBorder implements WidgetStateProperty<OutlinedBorder?> {
```

Comment:
Why does this need to change to `LinearBorder`?

Reply:
`OutlinedBorder` was an abstract class whereas `LinearBorder` is a concrete implementation, so we don't need to implement the abstract methods declared in `OutlineBorder`. I discussed it with Hans [here](https://github.com/flutter/flutter/pull/139431#discussion_r1457774261).